### PR TITLE
Saleor 4033 tests for plugins

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for plugins

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4033 Create tests for plugins](https://app.clickup.com/t/2549495/SALEOR-4033)

File: cypress/apiRequests/Customer.js [hunk 1]
```diff
@@ -90,6 +90,20 @@ export function customerRegistration({
   return cy.sendRequestWithQuery(mutation).its("body.data.accountRegister");
 }
 
+export function requestPasswordReset(email, channel) {
+  const mutation = `mutation{
```
Comment (dominik-zeglen) [inline cypress/apiRequests/Customer.js:94]: I'm wondering if there's a way to improve this and use `gql` tag

**Review by dominik-zeglen — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

File: src/plugins/components/PluginDetailsChannelsCard/PluginDetailsChannelsCardContent.tsx [hunk 2]
```diff
@@ -65,6 +65,7 @@ const PluginDetailsChannelsCardContent: React.FC<PluginDetailsChannelsCardProps>
         collection={plugin.channelConfigurations}
         renderItem={({ channel }) => (
           <div
+            data-tes-id="channel"
```
Comment (jwm0) [inline src/plugins/components/PluginDetailsChannelsCard/PluginDetailsChannelsCardContent.tsx:68]:
```suggestion
            data-test-id="channel"
```

File: src/plugins/components/PluginsList/PluginsList.tsx [hunk 3]
```diff
@@ -72,6 +72,7 @@ const PluginList: React.FC<PluginListProps> = props => {
           plugin =>
             plugin ? (
               <TableRow
+                data-test="id"
```
Comment (jwm0) [inline src/plugins/components/PluginsList/PluginsList.tsx:75]: test-id should be more precise
```suggestion
                data-test-id="plugin"
```

**Review by jwm0 — CHANGES_REQUESTED** [review]
After you've finished you should run `npm run test -- -u` to update snapshots (changes to `date-test-id`)

**Review by jwm0 — APPROVED** [review]

**Pull request merged**